### ISSUE TITLE
correction to fire_mort and impacts on cambial and crown mort diagnostics

### ISSUE THREAD

**Issue by rgknox** [NGEET/fates#977]
In the following line of code we calculate the fire mortality rate that we actually impose on the cohorts, as modified sum of the cambial and crown mortality:

https://github.com/NGEET/fates/blob/sci.1.61.0_api.25.0.0/fire/SFMainMod.F90#L1045-L1046

I'm not familiar with the basis for the equation used in Thonicke et al, by it looks like the intersection of the two mortalities is removed so that you don't kill tree's twice for different reasons, or something like that.  My question is, should we rescale the cambial and crown mortality terms after we calculate fire_mort, so that the history output reflects two rates that sum to resulting fire_mort?

**Comment by adrifoster** [comment]
Interesting idea, @rgknox. The original equations are from [Peterson and Ryan 1986](https://link.springer.com/article/10.1007/BF01867732), and the mortality rates are actually treated as probabilities (hence the calculation of their union as the final probability). So I guess it depends on what we want the output to represent?

**Comment by rgknox** [comment]
agreed, I don't see this as a bug, but more of what we want to represent in the diagnostic

**Comment by jkshuman** [comment]
agree with @adrifoster here, and thanks for adding the details for Peterson and Ryan 1986. and I see your point @rgknox that we want to talk about what the diagnostic is capturing.
